docs/resources/notification_group.md

```diff
@@ -0,0 +1,69 @@
+---
+page_title: "Mackerel: mackerel_notification_group"
+subcategory: "Notifications"
+description: |-
+
+---
+
+# Resource: mackerel_notification_group
+
+This resource allows creating and management of notification_group.
+
+## Example Usage
+
+```terraform
+resource "mackerel_notification_group" "example" {
+  name               = "Example notification group"
+  notification_level = "critical"
+
+  child_notification_group_ids = []
+  child_channel_ids = ["2vh7AZ21abc"]
+
+  monitor {
+    id           = "2qtozU21abc"
+    skip_default = false
+  }
+
+  service {
+    name = "Example-Service-1"
+  }
+  service {
+    name = "Example-Service-2"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `name` - (Required) The name of the notification group.
+* `notification_level` - The level of notification ("all" or "critical").
```

Comment:
I think it's better to describe about default value.

Reply:
Thank you. I added it. bff7725